frontend/src/component/Workspace/BtnExtandedAction.jsx

```diff
@@ -48,7 +48,7 @@ const ExtandedAction = props => {
         </div>
         */}
 
-        {props.userRoleIdInWorkspace >= 4 && (
+        {/* {props.userRoleIdInWorkspace >= 4 && (
```

Comment:
add the comment rules of algoo ("INFO - GB - data") and some explanation about why you are commenting this code with a reference to an issue or to something else